src/GraphQLParser/LexerContext.cs

```diff
@@ -116,94 +101,163 @@ public Token ReadNumber()
             return isFloat ? CreateFloatToken(start) : CreateIntToken(start);
         }
 
+        public Token ReadComment()
+        {
+            int start = _currentIndex;
+            char code = NextCode();
+
+            Span<char> buffer = stackalloc char[4096];
+            StringBuilder? sb = null;
```

Comment:
Perhaps a single StringBuilder can be allocated and shared for the entire context, so using pooling is not necessary.  (Or the context can grab a stringbuilder from a pool, and release it before returning.)

Reply:
It really is. This is actually a rare case when dealing with very large string literals. One can also take ROM from the pool. But this will complicate the code, I just didn’t want to make it even more complicated without an obvious need. Maybe later.

Reply:
I'm not using `StringBuilder` here because I like it. It stores a linked list of array objects inside itself, so there can still be a lot of memory allocations. Ideally, it would be working with one large array (behind ROM) from the pool. This array needs to be returned to the pool, so I will have to resort to `IMemoryOwner` services. It will need to be passed up the call stack, stored, etc.

Reply:
I think your latest changes with `ignorecomments` make this problem less important since default value for it is `true` in v7.